docs/MM_treatment_Episodes.md

```diff
@@ -0,0 +1,375 @@
+---
+layout: default
+title: Multiple myeloma treatment Episodes
+nav_order: 17
+description: "Drug Era Logic"
+
+---
+# Multiple myeloma treatment episodes in EPISODE. Creation logic
+
+## 
+
+## Background
+
+Drug data available in observational health databases can be abstracted
+on 4 principal levels: drug exposure (single drug administration or
+prescription), drug era (continuous drug administration or
+prescription), treatment regimen (drugs used in combination with fixed
+schedule), line of therapy (several regimens used consecutively united
+by one clinical intent). The first two levels exist in our OMOP common
+data model (CDM) datasets, and we need an effective way of capturing and
+storing lines and regimen.
+
+## Rules applied 
+
+### Rules for Regimen building
+
+1.  The first regimen starts as the first multiple myeloma
+    (MM)s-specific drug exposure after or on the day of the first MM
+    diagnosis (concept_id = 437233 [Multiple
+    myeloma](https://athena.ohdsi.org/search-terms/terms/437233))
+
+2.  All MM-specific drugs (see appendix 1) within 30 days of a regimen
+    start are considered part of same regimen (applicable to the first
+    and subsequent regimen)
+
+3.  The regimen ends if either:
+
+    1.  A new drug is added after the first 30 days
+
+    2.  Any drug from the first regimen is discontinued (not used for
+        more than 90 days), note if other drugs are discontinued within
+        30 days of this event, the last event within these 30 days is
+        considered the regimen end date.
+
+4.  A new regimen starts as either
+
+    1.  The day after the previous regimen end
+
+    2.  Start of the new drug era, if there was gap between previous
+        regimens
+
+5.  Steps 2 – 4 are repeated until all drug eras of drugs of interest
+    are covered
+
+6.  Add CAR-T regimens. Currently the CAR-T events are divided into 3
+    groups (subject of a discussion):
+
+a\) 'Apheresis Aph&CART' – when both Apheresis procedure and CAR-T
```

Comment:
@dimshitc Here these are a,b,c but the other sub-bullets are numbers. I would suggest choosing one to be consistent